content/questions/admin/question-006.md

```diff
@@ -0,0 +1,11 @@
+---
+question: "Which of the following are correct methods to generate and share a diagnostic file for GitHub Enterprise Server?"
+archetype: "questions"
+title: "Question 006"
+draft: false
+---
+
```

Comment:
Any documentation link  to this one?

Reply:
Nope sadly.

Reply:
This has some documentation about the tool `ghe-diagnostics` : https://docs.github.com/en/enterprise-server@3.10/support/contacting-github-support/providing-data-to-github-support#creating-and-sharing-diagnostic-files

Reply:
@chrheg  Thanks but I had to change this since it was a duplicate of Q5.